### DISCUSSION TITLE
I need to deploy the same next-auth enabled repository on two domains. How can I set NEXTAUTH_URL for both domains?

### DISCUSSION THREAD

**Discussion by ContraHacker** [nextauthjs/next-auth#3731] [Help]
### Question 💬

Greetings.

I need to deploy my NextJS website to two different domains. I'm using Vercel and I have pointed two domains to my project.

Next-auth requires the `NEXTAUTH_URL` env variable to be set equal to the domain name. With one domain, everything is GTG but how do I add another domain? I have saved the env variable to Vercel, it gets picked up and works fine.

### How to reproduce ☕️

1. Initialize next-auth with nextJS.
2. Use your domain's URL as NEXTAUTH_URL env variable.
3. Make sure to add all required env variables to your Vercel project.
3. Deploy to Vercel.
4. Add two (or more) domains to your project from within Vercel's project settings.

The first domain will work correctly, but any other domains won't allow you to authenticate via next-auth.

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by arvindell** [comment]
I have this same requirement, were you able to discover a solution?

**Reply by ContraHacker** [reply]
Nope! I ended up removing next-auth from one of the websites.
